### PR TITLE
Updated script to support Transifex API v3

### PR DESCRIPTION
**Pull request by ls-jf-gagnon**
The latest version of the Transifex client requires a different structure in the `.tx/config` file. Instead of being `<project-name>.<resource>`, the config needs to have the following structure `o:<organization>:p:<project-name>:r:<resource>`

### CODE REVIEW

**Review by ls-negar-jamalifard — APPROVED** [review]

**Comment by glambert** [comment]
Hey! Two things:

1. Did you test the workflow to validate it works? You can point to a [GitHub Action branch (or commit hash) as a version](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions)
2. What type of [version bump](https://semver.org/) is this change? You'll need to bump the version in `package.json` manually, then [run the release script](https://github.com/lightspeed/actions-push-translations/blob/master/release.sh) once the branch is merged.

Thanks for the contribution :+1:

File: src/entrypoint.ts
Comment (ls-georgios-siatras) [inline src/entrypoint.ts:60]: Super important critical thing, there is a double comment here 😅

**Review by ls-georgios-siatras — APPROVED** [review]

**Comment by ls-jf-gagnon** [comment]
> Hey! Two things:
> 
> 1. Did you test the workflow to validate it works? You can point to a [GitHub Action branch (or commit hash) as a version](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions)
> 2. What type of [version bump](https://semver.org/) is this change? You'll need to bump the version in `package.json` manually, then [run the release script](https://github.com/lightspeed/actions-push-translations/blob/master/release.sh) once the branch is merged.
> 
> Thanks for the contribution 👍

@glambert I ran a portion of the script locally, to validate that my changes worked. I was unable to to have the action running using the commit hash or the branch name :/

https://github.com/lightspeed-hospitality/hospitality-platform/actions/runs/3429350316/jobs/5714849026
https://github.com/lightspeed-hospitality/hospitality-platform/actions/runs/3430135391/jobs/5716711354

For the version change, it has to be a major bump since it's a breaking change. The version in the package.json is currently set at `1.0.0`, but the current release is tagged as `0.0.1`. I was thinking of keeping the version as is in the package.json and only tag the commit to `1.0.0`

The `release.sh` script can't be run on `master`, so I'll have to run it on this branch, once I get an approval, and I'll be able to merge the PR after

**Comment by glambert** [comment]
>The release.sh script can't be run on master, so I'll have to run it on this branch, once I get an approval, and I'll be able to merge the PR after

`Review has been requested on this pull request. It is not required to merge`. It's been approved nonetheless, you can go ahead and merge 🙂

**Pull request merged**
